tolumide-ng/EPIC_Mail
# #164910736 Integrate User Signup with Server

#### What does this PR do?
User can signup for EpicMail through the frontend

#### Description of Task to be completed?
- implement pitch-day feedback on signup page by removing carousel (distraction)
- Improve signup page form for easy usability
- enable fetch for signup button
- enable CORS on server
- the user can signup through this page
- saves successful fetch response in localStorage

#### How should this be manually tested?
Clone this repository, click the UI/index.html page and open on your browser, the user should be able to signup if all required details are entered correctly else the user should get a self-explaining error message if sign up was not successful


#### Any background context you want to provide?
Ensure that all required data are provided to avoid validation error


#### What are the relevant pivotal tracker stories?
#164910736
